README.md

```diff
@@ -92,15 +88,21 @@ If you're using Visual Studio Code for development, it's recommended that you:
+
+   > **Do not use any comments in your `api/.env` file** (it's technically possible to put comments in a .env file using the `#` character, but this causes problems for the API tunnel command we're going to use below)
+
 6. At the command prompt run `npm install` to install dependencies
 
 > **Note** Inside the `api` folder there are files `package.json` and `package-lock.json`. Every time either of these is modified, it is advised to repeat this step before running the project.
 
 7. Then run the command `npm start` to start the Volunteer App API server. You should see a message that says `Running scheduled cron jobs... ` and `Volunteer App API is listening on port <number> in development environment`.  Leave this terminal window open.
 
-8. Open another terminal window and in this new window run the command `npm run tunnel`. This 'tunnels' your local API server: makes it available externally so your app running in Expo Go can access it (using a free external service called [Serveo](https://serveo.net/)).  You should see a message saying your `Forwarding HTTP traffic from: https://xxxxxxxxx.serveo.net` -- this is the URL of your local API server, make a note of it as you'll need it in a minute.
+8. Open another terminal window and in this new window run the command `npm run tunnel-mac-linux` (if you are on a Mac or Linux) or `npm run tunnel-windows` (if you are on Windows). This 'tunnels' your local API server: makes it available externally so your app running in Expo Go can access it. You should see a message saying your `your url is: https://.............` -- this is the URL of your local API server, make a note of it as you'll need it in a minute.
+
+   > This URL should include the value you set for `API_TUNNEL_SUBDOMAIN` in your `api/.env` file, e.g. something like `https://nadia-bloggs-abc123def456ghi789.loca.lt`
 
-   > This URL should generally stay the same each time you run the tunnel command, but if you find it changes, you'll need to update the `STA_BASE_URL` value in your `app/.env` file (see step 11 below). (There are paid services or more complicated solutions we could use which guarantee a fixed URL, but for now this feels like the best solution that's easy to use and free for all devs in the team.)
+   > **If you get an error message `export: #: bad variable name`** that's because you have a comment (a line beginning with a `#`) in your `api/.env` file. Remove the commented line and try again.
 
```

Comment:
Or spaces. Doesn't like API_TUNNEL_SUBDOMAIN = "XXX" either.

Reply:
Good shout on the spaces -- thanks!